controllers/machineset_controller.go

```diff
@@ -375,9 +377,9 @@ func (r *MachineSetReconciler) getNewMachine(machineSet *clusterv1.MachineSet) *
 }
 
 // shouldExcludeMachine returns true if the machine should be filtered out, false otherwise.
-func shouldExcludeMachine(machineSet *clusterv1.MachineSet, machine *clusterv1.Machine) bool {
+func shouldExcludeMachine(machineSet *clusterv1.MachineSet, machine *clusterv1.Machine, logger logr.Logger) bool {
 	if metav1.GetControllerOf(machine) != nil && !metav1.IsControlledBy(machine, machineSet) {
-		klog.V(4).Infof("%s not controlled by %v", machine.Name, machineSet.Name)
+		logger.V(4).Info("machine is not controlled by machineset", "machine", machine.Name)
```

Comment:
I noticed that in a majority of the places `logger.Info` messages are capitalized. If it is intentional that logging messages are capitalized whereas the error messages follow the [Go style convention](https://github.com/golang/go/wiki/CodeReviewComments#error-strings), this should be `Machine is...`.